spec/integration/http_spec.rb

```diff
@@ -29,8 +29,19 @@
       end
     end
 
-    it 'inserts rack enforcer into the middleware stack' do
-      expect(middleware).to include 'Rack::SslEnforcer'
+    context 'if RAILS_ENV is not set to "test"' do
+      it 'inserts rack enforcer into the middleware stack' do
+        expect(middleware).to include 'Rack::SslEnforcer'
+      end
+    end
+
+    context 'if RAILS_ENV is set to "test"' do
+      before { ENV['RAILS_ENV'] = 'test' }
+      after { ENV['RAILS_ENV'] = nil }
```

Comment:
I'm surprised this works. Aren't we calling a rack script here by backticking out?